sympy/functions/combinatorial/tests/test_comb_numbers.py

```diff
@@ -457,6 +458,14 @@ def test_partition():
     assert partition(y).subs(y, 8) == 22
     raises(ValueError, lambda: partition(S(5)/4))
 
+def test__nT():
+       assert [_nT(i, j) for i in range(5) for j in range(i + 2)] == [
+    1, 0, 0, 1, 0, 0, 1, 1, 0, 0, 1, 1, 1, 0, 0, 1, 2, 1, 1, 0]
+       assert _nT(10, 5) == 7
+       assert _nT(100, 98) == 2
+       assert _nT(100, 100) == 1
+       assert _nT(10, 3) == 8
```

Comment:
When squashing, please add `assert [_nT(10, i) for i in range(11)] == [0, 1, 5, 8, 9, 7, 5, 3, 2, 1, 1]`

Reply:
okay

Reply:
I squashed everything down in my `nt` branch. If you download that and delete your partition branch and replace it with nt and then force push that here, everything will update.

Reply:
@smichr  the line for deleting partition is git -D partition but can you please tell what do you mean by replacing it with nt

Reply:
```
git checkout nt
git branch -D partition
git checkout -b partition
git branch -D nt
git push -f abhinav28071999 partition
```

Reply:
@smichr  I have done it. Please check it.